prql-compiler/tests/integration/main.rs

```diff
@@ -1,20 +1,43 @@
 // Re-enable on windows when duckdb supports it
 // https://github.com/wangfenjin/duckdb-rs/issues/62
 #![cfg(not(any(target_family = "windows", target_family = "wasm")))]
+// TODO enable it for all OS
+#![cfg(target_os = "linux")]
+#![cfg(feature = "test-external-dbs")]
```

Comment:
(fine to leave for another PR)

I was thinking we would have something like:

```rust
fn get_connections() -> Vec<dyn Connection> {
   #[cfg(feature = "test-external-dbs")]
   vec![...]

   #[cfg(not(feature = "test-external-dbs"))]
   vec![...]
}
```

(not exact code, probably they need to be initialized etc)

Totally fine to have a different design — you're the decider — would be great to always run the inmemory DBs regardless of the feature, though.

WDYT?

Reply:
Good idea.

Reply:
Should the test fail, if the feature is active and the DBs are not running? Probably yes

Reply:
Yes, for sure